config.sample

```diff
@@ -13,7 +13,9 @@ location=location_here
 # metasmoke_key=gibberish_here
 
 # Set GitHub keys
-# git_username=SmokeDetector
-# git_useremail=smokey@erwaysoftware.com
```

Comment:
These should not be removed. They are actively used on every commit and merge.

Reply:
They're not in the `sample-config` in Keybase. Are they optional and defaulting to the values in `git config` or something like that?

Reply:
Ah, they [have default values](https://github.com/Charcoal-SE/SmokeDetector/blob/e3e35ba4ac0642f72e42e2a20f77da857d629b4a/globalvars.py#L246). However, it might be worth keeping them in for the sake of documentation.

Reply:
I disagree with the removal of these keys, leave them there for documentation purposes.  This needs adjusted to *not* remove those lines, please.

Reply:
Yes, the config.sample file in the repository is intended as documentation. It shows what information is needed to set up the bot for someone running SD without the SmokeDetector credentials. These values need to be set whenever the bot is run and is making commits to `git`/GitHub using an account other than SmokeDetector. The default values are fine to be used by people running authorized instances.